lib/cartopy/mpl/slippy_image_artist.py

```diff
@@ -53,7 +53,7 @@ def draw(self, renderer, *args, **kwargs):
 
         ax = self.axes
         window_extent = ax.get_window_extent()
-        [x1, y1], [x2, y2] = ax.viewLim.get_points()
+        [x1, y1], [x2, y2] = ax.viewLim.get_points().tolist()
```

Comment:
Can you add a comment here saying something along the lines of "we need to extract to a list to get native Python datatypes rather than Numpy datatypes because not all fetch_raster implementers handle numpy datatypes properly"

Reply:
Comment added.